src/number.rs

```diff
@@ -1,7 +1,48 @@
 // Exports of <CoreFoundation/CFNumber.h>
 
+use libc::c_void;
+
 use base::*;
 
+#[repr(C)]
+struct __CFBoolean;
+pub type CFBooleanRef = __CFBoolean;
+
+extern "C" {
+    pub fn CFBooleanGetTypeID() -> CFTypeID;
+    pub fn CFBooleanGetValue(boolean: CFBooleanRef) -> Boolean;
+}
+
+pub type CFNumberType = CFTypeID;
```

Comment:
The values of the enum would still be defined as `CFNumberType` though, correct?
```rust
pub const kCFNumberSInt8Type: CFNumberType = 1;
```